cogs/currency.py

```diff
@@ -6,28 +6,59 @@
 class Currency(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.c = sqlite3.connect('currency.db')
-        self.connect = self.c.cursor()
+        self.conn = sqlite3.connect('currency.db')
+        self.c = self.conn.cursor()
+
+    def modify_balance(self, author, target, amount):
+        self.c.execute("UPDATE CURRENCY SET balance = balance + ? where UID = ?", [amount, author])
+        self.c.execute(
+            "UPDATE CURRENCY SET balance = balance + ? where UID = ?", [-amount, target])
+        self.conn.commit()
 
     @commands.group()  # %credits
     async def credits(self, ctx):
         if ctx.invoked_subcommand is None:
-            await ctx.send('You\'re currently missing parameters in your request.\n\n' +
-                           'In order to properly respond we need the following information from your query.\n\n' +
-                           '(1)!credits (2)Requested Operation (3)Targeted User (4)Credit Amount\n' +
-                           'Requested operations are [Give, Request, Destroy, Rob]')
```

Comment:
I believe this should still live somewhere. You're really over reaching for just adding db functionality in this PR.